internal/fileutils/fileutils.go

```diff
@@ -713,34 +717,25 @@ func CopyFilesFromAssets(assetDir, dst string) error {
 				return errs.Wrap(err, "MkdirUnlessExists %s failed", destPath)
 			}
 
-			err = CopyFilesFromAssets(srcPath, destPath)
+			err = CopyFilesDirReader(reader, srcPath, destPath)
 			if err != nil {
 				return errs.Wrap(err, "CopyFiles %s:%s failed", srcPath, destPath)
 			}
-		default:
-			err := CopyAsset(srcPath, destPath)
+		case os.ModeSymlink:
+			err := CopySymlink(srcPath, destPath)
 			if err != nil {
-				return errs.Wrap(err, "CopyFile %s:%s failed", srcPath, destPath)
+				return errs.Wrap(err, "CopySymlink %s:%s failed", srcPath, destPath)
+			}
+		default:
+			if entry.Name() == assets.PlaceholderFileName {
+				continue
```

Comment:
This is opinionating it towards assets again. Perhaps provide an argument where you can pass in a custom filter function.